movie/src/components/Button/Button.jsx

```diff
@@ -0,0 +1,22 @@
+import { Component } from "react";
+
+export default class Button extends Component {
```

Comment:
Why do you use a class component here?

Reply:
it's a examples component with ErrorBuundary